src/trace_processor/perfetto_sql/stdlib/android/input.sql

```diff
@@ -149,7 +149,7 @@ JOIN actual_frame_timeline_slice AS app
   ON surface_flinger.display_frame_token = app.display_frame_token
   AND surface_flinger.id != app.id
 WHERE
-  surface_flinger.surface_frame_token = 0 AND app.present_type != 'Dropped Frame';
```

Comment:
Please add a test.

Reply:
Added a diff_test in 4671d07bf36d5e9ff6e5efba1102baa60830a9dd